gems/aws-sigv4/sig/credentials.rbs

```diff
@@ -0,0 +1,30 @@
+module Aws
+  module Sigv4
+    class Credentials
+      def initialize: (
+        access_key_id: String,
+        secret_access_key: String,
+        session_token?: String?
+      ) -> void
+
+      attr_reader access_key_id: String
+      attr_reader secret_access_key: String
+      attr_reader session_token: String?
+
+      def set?: () -> bool
+    end
+
+    class StaticCredentialsProvider
```

Comment:
ditto.
https://github.com/aws/aws-sdk-ruby/blob/8c6f65feb0500775b50153e0c24a56381509c873/gems/aws-sigv4/lib/aws-sigv4/credentials.rb#L43-L46

Reply:
Sorry, I deleted it.
a72b3bcfb6b8ab60ae2fd5cb968b861c7f02fe84